frontend/src/components/contactsList.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react'
+import { CreateAContactForm } from './contactsForm'
+import axios from 'axios'
+
+interface Contact {
```

Comment:
is this copied directly from prisma? shouldn't we just import `Contact` from prisma instead?